celer/PN_logreg.pyx

```diff
@@ -102,19 +102,19 @@ def newton_celer(
     cdef bint positive = 0
 
     for t in range(max_iter):
-        p_obj = primal(LOGREG, alpha, Xw, y, w, weights_pen)
+        p_obj = primal(LOGREG, alpha, 1.0, Xw, y, w, weights_pen)
```

Comment:
it's more explicit if you define `cdef floating l1_ratio=1` above and then use `l1_ratio` here and in the rest of the function.

It also makes your future life easier if you want to implement enet regularization for PN_logreg one day.